charts/posthog/values.yaml

```diff
@@ -2426,7 +2426,7 @@ prometheus:
                 description: "The `session_recording_events_dlq` topic offset has increased over the past 5 minutes."
 
             - alert: GraphileJobExecutionLag
-              expr: (max by(task_identifier) (posthog_celery_graphile_lag_seconds{task_identifier!="bufferJob"})) > 900
```

Comment:
Heya, we still want the `by(task_identifier)`, otherwise a second type lagging will not trigger another alert